libportability-gfx/src/impls.rs

```diff
@@ -62,7 +62,7 @@ pub extern "C" fn gfxCreateInstance(
         let _ = env_logger::try_init();
     }
 
-    let mut backend =
+    let backend =
```

Comment:
there is a mutation of the backend below, inside the Metal arm, so this isn't going to build on macos

Reply:
Ah, good thing CI can check those, since I can't. Alright, in that case, I will just disable warning for that statement, along with note why.